db/special_docs.go

```diff
@@ -24,10 +24,21 @@ func (db *Database) GetSpecial(doctype string, docid string) (Body, error) {
 	}
 
 	body := Body{}
-	_, err := db.Bucket.Get(key, &body)
-	if err != nil {
-		return nil, err
+
+	if doctype == "local" {
+		rawDocBytes, _, err := db.Bucket.GetAndTouchRaw(key, base.SecondsToCbsExpiry(int(db.DatabaseContext.Options.LocalDocExpirySecs)))
```

Comment:
Can we do a regular Get if LocalDocExpirySecs is zero?

Reply:
Updated if condition to if doctype == "local" && db.DatabaseContext.Options.LocalDocExpirySecs > 0 {